src/Files.App/Shell/ContextMenuItem.cs

```diff
@@ -19,17 +16,12 @@ public void Dispose()
 
 		protected virtual void Dispose(bool disposing)
 		{
-			if (disposing)
+			if (disposing && SubItems is not null)
```

Comment:
Why was the null protection removed?

Reply:
The behavior has not changed. I just used a single if instead of 2 nested ones.